plugins-extra/query_enhancements/server/routes/index.ts

```diff
@@ -91,4 +91,75 @@ export function defineRoutes(
       }
     }
   );
+
+  // sql async jobs
+  router.post(
+    {
+      path: `/api/sqlasyncql/jobs`,
+      validate: {
+        body: schema.object({
+          query: schema.object({
+            qs: schema.string(),
+            format: schema.string(),
+          }),
+          df: schema.any(),
```

Comment:
i think we can let the df.meta.sessionId keep responsible for this.

if it has a session id then we just pull it and use it.